api/cluster.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/raintank/metrictank/api/middleware"
+	"github.com/raintank/metrictank/api/models"
+	"github.com/raintank/metrictank/api/rbody"
+	"github.com/raintank/metrictank/cluster"
+)
+
+func (s *Server) getNodeStatus(ctx *middleware.Context) {
+	rbody.WriteResponse(ctx, rbody.NewJsonResponse(200, cluster.ThisNode, ""))
+}
+
+func (s *Server) setNodeStatus(ctx *middleware.Context, status models.NodeStatus) {
+	cluster.ThisNode.SetPrimary(status.Primary)
+	rbody.WriteResponse(ctx, rbody.NewJsonResponse(200, "ok", ""))
+}
+
+func (s *Server) appStatus(ctx *middleware.Context) {
+	if cluster.ThisNode.IsReady() {
+		rbody.WriteResponse(ctx, rbody.NewJsonResponse(200, "ok", ""))
```

Comment:
previous code would return `OK` with `Content-Type: text/plain`
now we will return: `"ok"` (lowercase and quoted) `content-type: application/json`

why do we even a need json encoder here? it just adds cruft imho